discovery_engine_core/core/src/storage/migrations/20220727161140_view_time.sql

```diff
@@ -0,0 +1,26 @@
+--  Copyright 2022 Xayn AG
+--
+--  This program is free software: you can redistribute it and/or modify
+--  it under the terms of the GNU Affero General Public License as
+--  published by the Free Software Foundation, version 3.
+--
+--  This program is distributed in the hope that it will be useful,
+--  but WITHOUT ANY WARRANTY; without even the implied warranty of
+--  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--  GNU Affero General Public License for more details.
+--
+--  You should have received a copy of the GNU Affero General Public License
+--  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+CREATE TABLE IF NOT EXISTS ViewTimes(
```

Comment:
❓ the other tables in the database use singular case for the names, should this be done here as well?
```suggestion
CREATE TABLE IF NOT EXISTS ViewTime(
```

Reply:
It's plural because it has multiple entries per document, while each other has one entry per document.